numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -2836,10 +2836,7 @@ def add_ptx(self, ptx, name):
     def add_cu(self, cu, name):
         """Add CUDA source in a string to the link. The name of the source
         file should be specified in `name`."""
-        with driver.get_active_context() as ac:
-            dev = driver.get_device(ac.devnum)
-            cc = dev.compute_capability
-        ptx, log = nvrtc.compile(cu, name, cc)
+        ptx, log = nvrtc.compile(cu, name, self.cc)
```

Comment:
I think the CC is always specified when creating a linker now, so to fix the CI issue that resulted from this addition, I think adding `self.cc = cc` to `CtypesLinker.__init__()` should be safe.